projects/auth0-angular/package.json

```diff
@@ -26,8 +26,8 @@
   },
   "homepage": "https://github.com/auth0/auth0-angular#readme",
   "peerDependencies": {
-    "@angular/common": "^10.0.2",
-    "@angular/core": "^10.0.2"
+    "@angular/common": ">=9 <12",
+    "@angular/core": ">=9 <12"
```

Comment:
This might be better written as `<=11` to save some confusion. When I first read it I was like "But 12 isn't out ye... oooh"

Reply:
I couldnt find proper documentation around this but I think `<=11` means `<=11.0.0`, which led me to consider `<=^11` which I couldnt find documentation on whether or not this is supported.

So <12 was the one I am the most confident about that it does what we want.

Reply:
That's fair enough, just raising the question :)

Reply:
https://jubianchi.github.io/semver-check/#/%3E%3D9%20%3C%3D11/11.20.0

Looks like `>=9 <=11` works.